python-sdk/docs/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 1.5.0a1
+
+### Feature:
+- Add `load_options` param to accept `pandasOptions` in `load_file` operator [#1466](https://github.com/astronomer/astro-sdk/pull/1466)
```

Comment:
Is this `pandasOption`? -- why camel case?

Reply:
It's a class name

Reply:
`pandasOption` is a class

Reply:
ahhh..should be `PandasLoadOptions` will fix in beta/minor release

Reply:
isn't this has been already released in 1.4.0 https://github.com/astronomer/astro-sdk/blob/main/python-sdk/docs/CHANGELOG.md#feature-1 ?

Reply:
@kaxil `pandasOption` is a class and we can pass it in `load_options ` so it should be in camel case, WDYT?

Reply:
@pankajastro it's added in the milestone for 1.5.0 - https://github.com/astronomer/astro-sdk/milestone/27?closed=1

Reply:
@pankajastro it is `PandasOption` :) not `pandasOption`

Reply:
We don't use camelCase in Python -- it is popular in Go/Java/JS/C#

Reply:
I'll fix this in a minor release/beta release.

Reply:
ohh I typed it wrong :man_facepalming:. The class name is `PandasOption` right? The changelog text should change accordingly